src/redux/user-store.ts

```diff
@@ -11,8 +11,8 @@
  */
 /*
     at src/redux/store.ts:12:33
```

Comment:
remove this debug comment, there no reason to have a outdated stacktrace in code